Makefile

```diff
@@ -112,3 +112,44 @@ merge_esmero:
 rebase_esmero_webform_strawberryfield:
 	cd web/modules/contrib/webform_strawberryfield && git stash && git checkout $(WEBFORM_SBF_BRANCH) && git rebase upstream/$(WEBFORM_SBF_BRANCH) && git stash apply
 
+build_linux:
+	cp docker-compose-linux.yml docker-compose.yml
+	sed 's/version: '\''3.5'\''/version: '\''3.7'\''/' docker-compose.yml
+	sed 's/image: mysql:8.0.22/image: mysql:5.7.33/' docker-compose.yml
+	docker-compose up -d
+	sudo chown -R 100:100 persistent/iiifcache
+	sudo chown -R 8983:8983 persistent/solrcore
+	docker exec -t $(APACHE_CONTAINER) bash -c "chown -R www-data:www-data private"
+	docker-compose exec -T mc bash -c "mc alias set minio http://minio:9000 minio minio123"
+	docker-compose exec -T mc bash -c "mc mb minio/archipelago"
+	docker-compose exec -T mc bash -c "mc ls minio"
+	docker exec -t -w /var/www/html $(APACHE_CONTAINER) bash -c "COMPOSER_MEMORY_LIMIT=-1 composer install -o --prefer-dist --no-interaction"
+	#docker-compose exec -T -w /var/www/html php bash -c "chown -R :www-data /var/www/html/web"
+	docker exec -ti $(APACHE_CONTAINER) bash -c 'scripts/archipelago/setup.sh'
+	docker exec -ti $(APACHE_CONTAINER) bash -c "cd web;../vendor/bin/drush -y si --verbose config_installer config_installer_sync_configure_form.sync_directory=/var/www/html/config/sync/ --db-url=mysql://root:esmerodb@esmero-db/drupal8 --account-name=admin --account-pass=archipelago -r=/var/www/html/web --sites-subdir=default --notify=false install_configure_form.enable_update_status_module=NULL install_configure_form.enable_update_status_emails=NULL;drush cr;chown -R www-data:www-data sites;"
+	docker exec -ti $(APACHE_CONTAINER) bash -c 'drush ucrt demo --password="demo"; drush urol metadata_pro "demo"'
+	docker exec -ti $(APACHE_CONTAINER) bash -c 'drush ucrt jsonapi --password="jsonapi"; drush urol metadata_api "jsonapi"'
+	docker exec -ti $(APACHE_CONTAINER) bash -c 'scripts/archipelago/deploy.sh'
```

Comment:
This is all redundant and can be removed.

Reply:
Having reviewed again, yes some elements are redundant however this isn't

```bash
	docker-compose exec -T mc bash -c "mc alias set minio http://minio:9000 minio minio123"	docker-compose exec -T mc bash -c "mc mb minio/archipelago"	docker-compose exec -T mc bash -c "mc ls minio"
```

Reply:
I think we just need to slip that in now after line 122 in the Makefile

Reply:
Hmm so build_linux is now removed. Where should these lines go? `build_dev_linux` and `build_archipelago_contributor_linux` ?

Reply:
Okay I've updated those two locations and removed older steps with making local minio directories? The idea is that minio is used as a volume to avoid perms issues and that the minio command line container is created to programatically create the minio materials etc.

Reply:
`build_linux` section is all removed now.